region/brinstar/green.json

```diff
@@ -2082,6 +2143,47 @@
         {
           "from": 1,
           "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Beetom Farm",
+                  "notable": false,
+                  "requires": [
+                    {"resetRoom": {
+                      "nodes": [1, 2],
+                      "mustStayPut": false
+                    }},
+                    {"or": [
+                      "ScrewAttack",
+                      {"and": [
+                        "Ice",
+                        "Morph",
+                        {"or": [
+                          "Bombs",
+                          {"and": [
+                            "PowerBomb",
+                            {"or": [
+                              {"ammo": {"type": "Missile", "count": 1}},
+                              {"ammo": {"type": "Super", "count": 1}},
+                              {"ammo": {"type": "PowerBomb", "count": 1}}
+                            ]}    
+                          ]}
+                        ]}
+                      ]}
+                    ]},
+                    {"refill": ["Energy", "PowerBomb"]}
```

Comment:
Beetoms aren't known for their energy drops.  Is 4 smalls at a time ok?

Missile/Super would give a power bomb to use to kill the rest, and then refill more pbs.  That's a neat interaction.

Reply:
Right, the "Energy" refill was a mistake here, removed that now.